src/DynamicMvvm.Tests/ViewModel/ViewModelBaseCommandsTest.cs

```diff
@@ -76,6 +76,20 @@ public async Task It_Gets_From_Task_T()
 			receivedParameter.Should().Be(parameter);
 		}
 
+		[Fact]
+		public async Task It_Get_From_Item_Task()
+		{
+			var receivedParameter = default(TestEntity);
+			var viewModel = new ViewModelBase(serviceProvider: _serviceProvider);
+
+			var command = viewModel.GetItemCommandFromTask<TestEntity>(async (ct, p) => receivedParameter = p);
```

Comment:
You should have a test that validates that all instances returned by this method are different, even when using the same name.